components/logins/src/logins-managed-store.udl

```diff
@@ -0,0 +1,148 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+namespace logins {
```

Comment:
we are very very close to being able to use procmacros, which would remove the need for all this duplication :)